hangman.js

```diff
@@ -1 +1,72 @@
-console.log("welcome to hangman");
\ No newline at end of file
+//Test area
+console.log("welcome to hangman");
+alert("Hello!");
+
+
+//Do I want to create an object?
+
+
+
+var secretWord;
+var playerGuess;
+var score = [];
+var incorrectGuesses = 5;
+
+//Prompt player1 to enter their secret word
+var startGame = function() {
+  secretWord = prompt("Please enter the secret word.");
+  secretWord=secretWord.split("");
+  console.log(secretWord);
+};
+//The secret word needs to be an array with indexes for each invidual letter
+//Take input text and convert to array with each letter as an index.
+
+
+//Prompt player2 to guess letter
+  //This window should show dashes that correspond to the number of letters in the secret word.  It should also keep track of the amount of incorrect guesses remaining (begin with 5).
+  //Player2 inputs letter
+var playerTurn = function() {
+    playerGuess = prompt("Guess a letter." + score + incorrectGuesses);
+for (i = 0; i < secretWord.length; i ++) {
```

Comment:
Pay attention to indentation here. Not only makes it more readable for me but gives you a better idea of where functions, conditional statements, etc. begin and end!